viewing_party/party.py

```diff
@@ -1,23 +1,166 @@
-# ------------- WAVE 1 --------------------
+from pprint import pprint

```

Comment:
An extraneous import? Probably auto-imported from VSCode. These unused imports should not be committed. I recommend turning off that setting in VS Code.